AdyenComponents/UPI/UPIComponent.swift

```diff
@@ -0,0 +1,217 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import UIKit
+
+/// The flow types for UPI component.
+internal enum UPIFlowType: Int {
```

Comment:
We can nest this into UPIComponent so it's not polluting the namespace
`UPIComponent.FlowType` etc

Reply:
Fixed. 064576ad38d6b20fdbad506aa73c0f791f3f91bc